src/main/config/log4j2.properties

```diff
@@ -0,0 +1,7 @@
+status=error
+dest=err
+
+appender.console.type = Console
```

Comment:
You can use this... I think YML or JSON is more readable, though.

Reply:
I agree, but this kept it the closest to what we already had. I'll explore JSON or YAML in the future